core/templates/services/utils.service.ts

```diff
@@ -27,6 +27,10 @@ export class UtilsService {
   // The function here is to check whether the argument is empty or not. So, we
   // cannot have any specific type defined for the argument and the argument
   // is given a generic type of Object.
+  /**
+   * @param {Object} obj - the object to be checked
```

Comment:
Here and elsewhere, add a period in end.

Reply:
I have added the periods.